modules/gds/pages/featurizer.adoc

```diff
@@ -17,20 +17,25 @@ Prints the information for algorithms within the specified category.
 
 
 == installAlgorithm()
-`installAlgorithm(query_name: str, query_path: str = None) -> str`
+`installAlgorithm(query_name: str, query_path: str = None, global_change: bool = False) -> str`
 
 Checks if the query is already installed. 
 If the query is not installed, it installs the query and changes the schema if an attribute needs to be added.
 
 [discrete]
 ==== Parameters:
-* `query_name (str)`: The name of query to be installed.
+query_name (str): 
+The name of query to be installed
 * `query_path (str)`: If using a custom query, the path to the `.gsql` file that contains the query.
 Note: you must have the `query_name` parameter match the name of the query in the file.
 
+* `global_change (bool)`: False by default. Set to true if you want to run `GLOBAL SCHEMA_CHANGE JOB`. For Algorithms that are not schema free we need to specify this argument.
```

Comment:
No need to capitalize "algorithms."

Maybe reword to "This argument is required for algorithms that are not schema-free."